src/daemon/rpmostreed-transaction-types.cxx

```diff
@@ -2279,6 +2279,7 @@ refresh_md_transaction_execute (RpmostreedTransaction *transaction, GCancellable
   rpmostree_context_configure_from_deployment (ctx, sysroot, cfg_merge_deployment);
 
   /* don't even bother loading the rpmdb */
+  // TODO: Define the behaviour for when refresh-md is called with respect to filelists
```

Comment:
This is what I have in mind for the behaviour of `refresh_md_transaction_execute()` with respect to setting `DNF_CONTEXT_SETUP_SACK_FLAG_SKIP_FILELISTS`. I think that if filelists have already been downloaded (ie. by previously running `rpm-ostree install /usr/bin/htop`), the refresh_md function should have `DNF_CONTEXT_SETUP_SACK_FLAG_SKIP_FILELISTS` disabled. If no filelists are downloaded on the system yet, then `DNF_CONTEXT_SETUP_SACK_FLAG_SKIP_FILELISTS` should be enabled. Therefore, changes to this function would just require some way of detecting if filelists have been downloaded on the system. Do you guys have any thoughts on this?